packages/devtools_app/lib/src/timeline/timeline_service.dart

```diff
@@ -129,18 +128,13 @@ class TimelineService {
           '$threadNames');
     }
 
-    timelineController.frameBasedTimeline.processor =
-        FrameBasedTimelineProcessor(
-      uiThreadId: uiThreadId,
-      gpuThreadId: gpuThreadId,
-      timelineController: timelineController,
-    );
-
-    timelineController.fullTimeline.processor = FullTimelineProcessor(
-      uiThreadId: uiThreadId,
-      gpuThreadId: gpuThreadId,
-      timelineController: timelineController,
-    );
+    for (var timeline in timelineController.timelines) {
```

Comment:
great to see this code simplified a bit. it will really help when someone else needs to modify this code.